OpenSim/Simulation/SimbodyEngine/Joint.h

```diff
@@ -529,21 +495,28 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Joint, ModelComponent);
         _slaveBodyForChild = slaveForChild;
     }
 
-private:
-    //=========================================================================
+    //==========================================================================
     // DATA
-    //=========================================================================
+    //==========================================================================
+protected:
+    // Specifies the direction of the Joint in the multibody tree: parent->child
+    // (isReversed is false; default) or child->parent (isReversed is true). The
+    // Joint's transform and coordinates maintain a parent->child sense even if
+    // the joint is reversed.
+    bool isReversed = false;
```

Comment:
> I am going to remove getIsReversed() in part 2 when I improve the tests (getIsReversed() is used only by testJoints). The fact that joints can be reversed internally should be completely hidden from the user. I will add your note ("Before OpenSim 4.0, …") to CHANGELOG.

Okay sounds good.

> We haven't been consistent with updating CHANGELOG

I know...I would like to create a [pull request template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) that reminds us to update the CHANGELOG, but I can't get over the fact that we would have to add another file or directory to the root of the project.